sshdconfig/src/get.rs

```diff
@@ -82,3 +85,68 @@ fn get_default_shell() -> Result<(), SshdConfigError> {
 fn get_default_shell() -> Result<(), SshdConfigError> {
     Err(SshdConfigError::InvalidInput(t!("get.windowsOnly").to_string()))
 }
+
+fn get_sshd_settings(input: Option<&String>) -> Result<(), SshdConfigError> {
+    let config = extract_metadata_from_input(input)?;
+    let mut exclude_defaults = false;
+    let mut args = None;
+    if !config.metadata.is_empty() {
+        if let Some(value) = config.metadata.get("defaults") {
+            if let Value::Bool(b) = value {
+                exclude_defaults = !b;
+            } else {
+                return Err(SshdConfigError::InvalidInput(t!("get.defaultsMustBeBoolean").to_string()));
+            }
+        }
+        if let Some(filepath) = config.metadata.get("filepath") {
+            if let Value::String(path) = filepath {
+                args = Some(
+                    SshdCommandArgs {
+                        filepath: Some(path.clone()),
+                        additional_args: None,
+                    }
+                );
+            } else {
+                return Err(SshdConfigError::InvalidInput(t!("get.filepathMustBeString").to_string()));
+            }
+        }
+    }
```

Comment:
I think handling of metadata should be a separate function and put the results into a struct